src/Bundle/DependencyInjection/Compiler/FosRestPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\DependencyInjection\Compiler;
+
+use FOS\RestBundle\FOSRestBundle;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class FosRestPass implements CompilerPassInterface
```

Comment:
Would be cool to make more explicit name, like

```suggestion
final class UnregisterDefinitionsOnNonInstalledFOSRestBundlePass implements CompilerPassInterface
```

I'm not suggesting this particular name though, but stating intention in name (Unregister) and more full vendor name (FOSRestBundle) looks better to me.

Reply:
I agree and I've changed that with sth between previous name and yours.